installer/src/main/lib/tool.properties

```diff
@@ -0,0 +1,8 @@
+connect.timeout=120000
+activate.timeout=180000
+deploy.timeout=180000
+redeploy.timeout=180000
+undeploy.timeout=180000
+start.application.timeout=180000
+stop.application.timeout=180000
+set.server.groups.timeout=30000
```

Comment:
add newline at the end of the file (more portable)

Reply:
done